qiskit_experiments/data_processing/sklearn_discriminators.py

```diff
@@ -99,3 +103,72 @@ def from_config(cls, config: Dict[str, Any]) -> "SkLDA":
                 setattr(lda, name, value)
 
         return SkLDA(lda)
+
+
+class SkCLF(BaseDiscriminator):

```

Comment:
Can you add a test for this class that does the following: (i) setup a `SkCLF` discriminator (ii) train it (iii) predict something with it (iv) serialize it into a config (v) deserialize a new `SkCLF` instance from the config of (iv) and use this deserialized instance to predict the same data from step (iii) and assert that the results are identical. This will ensure round-trip serialization works.

Reply:
Added in commit [12d3f6a](https://github.com/Qiskit/qiskit-experiments/pull/936/commits/12d3f6a234f4ec74362878eab71c29b96b6a16b3).